src/Cache/_ScsDataClient.php

```diff
@@ -521,7 +520,6 @@ public function dictionarySetBatch(string $cacheName, string $dictionaryName, ar
             validateCacheName($cacheName);
             validateDictionaryName($dictionaryName);
             validateItems($items);
-            validateFieldsKeys($items);
```

Comment:
just curious what the deal with this one was

Reply:
Hmm sorry I thought I added it back after trying to figure out why it failed. I'm adding this back.